friends/src/App.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import './App.css';
+import axios from 'axios';
+import Friend from './components/friends'
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      friends: [],
+      name: '',
+      age: '',
+      email: '',
+      id: ''
+    }
+  }
+
+  componentDidMount() {
+    axios
+      .get('http://localhost:5000/friends')
+      .then(response => {
+        console.log(response)
+        this.setState({ friends: response.data })
+      })
+  }
+  onChangeHandler = e => {
+    const name = e.target.name;
+    this.setState({ [name]: e.target.value })
+  }
+
+  addFriendHandler = (e) => {
```

Comment:
This is just another way of passing your state into `axios.post`. Now your second argument would look like `{name, age, email }`. You can read more about destructuring `state` in `React` [here](https://wesbos.com/destructuring-objects/)
```suggestion
  addFriendHandler = (e) => {
  const { name, age, email } = this.state;
```